tutorials/roostats/rs401c_FeldmanCousins.py

```diff
@@ -0,0 +1,95 @@
+## \file
+## \ingroup tutorial_roostats
+## \notebook
+## Produces an interval on the mean signal in a number counting experiment with known background using the
+## Feldman-Cousins technique.
+##
+## Using the RooStats FeldmanCousins tool with 200 bins
+## it takes 1 min and the interval is [0.2625, 10.6125]
+## with a step size of 0.075.
+## The interval in Feldman & Cousins's original paper is [.29, 10.81] Phys.Rev.D57:3873-3889,1998.
+##
+## \macro_image
+## \macro_output
+## \macro_code
+##
+## \date July 2022
+## \authors Artem Busorgin, Kyle Cranmer (C++ version)
+
+import ROOT
+
+# to time the macro... about 30 s
+t = ROOT.TStopwatch()
+t.Start()
+
+# make a simple model
+x = ROOT.RooRealVar("x", "", 1, 0, 50)
+mu = ROOT.RooRealVar("mu", "", 2.5, 0, 15)  # with a limit on mu>=0
+b = ROOT.RooConstVar("b", "", 3.0)
+mean = ROOT.RooAddition("mean", "", [mu, b])
+pois = ROOT.RooPoisson("pois", "", x, mean)
+parameters = {mu}
+
+# create a toy dataset
+data = pois.generate({x}, 1)
+data.Print("v")
+
+dataCanvas = ROOT.TCanvas("dataCanvas")
+frame = x.frame()
+data.plotOn(frame)
+frame.Draw()
+dataCanvas.Update()
+
+w = ROOT.RooWorkspace()
+modelConfig = ROOT.RooStats.ModelConfig("poissonProblem", w)
+modelConfig.SetPdf(pois)
+modelConfig.SetParametersOfInterest(parameters)
+modelConfig.SetObservables({x})
+w.Print()
+
+# show use of Feldman-Cousins
+fc = ROOT.RooStats.FeldmanCousins(data, modelConfig)
+fc.SetTestSize(0.05)  # set size of test
+fc.UseAdaptiveSampling(True)
+fc.FluctuateNumDataEntries(False)  # number counting analysis: dataset always has 1 entry with N events observed
+fc.SetNBins(100)  # number of points to test per parameter
+
+# use the Feldman-Cousins tool
+interval = fc.GetInterval()
+
+# make a canvas for plots
+intervalCanvas = ROOT.TCanvas("intervalCanvas")
```

Comment:
@guitargeek when running this with master, it gets stuck on creating the second Canvas - "intervalCanvas".

I can remove this line and was able to get two options working:
1. SaveAs the data point plot right after we draw it, then the histogram reuses the "dataCanvas". Then calling SaveAs again to save the histogram at the end.
2. Splitting the "dataCanvas" into two pads, and display both plots in the same canvas and save that.

Reply:
Hi, thanks for the updates! I don't get this problem fortunately. Maybe it has to do something with your environment? Which operating system are you using? If you can reproduce this "getting stuck" in a simple way, it would be nice if you could open a separate GitHub issue for such, such that the experts can investigate :+1:

In any case, the issue is not caused by your translation, and if the tutorials are all running on the CI, this PR can be merged.

Reply:
That is fortunate. I will try to reproduce it with a simpler example, and create an issue for it.

I'm running on Windows 10 Enterprise 20H2. Built from source using VS 2019